New log file opened at Tue Apr 05 17:03:49 CEST 2016 by 8036@kamyh-surface
Taking next connection to: 51665
New client attempting to connect on port: 51666-51665
  Address = /10.128.25.213:51665
This client on port 51666 has uuid 3c8d84e3-5e14-4ad9-b687-7e97b6c33e19 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-3c8d84e3-5e14-4ad9-b687-7e97b6c33e19
Taking next connection to: 51665
New client attempting to connect on port: 51667-51665
  Address = /10.128.25.213:51665
Handshake exception on socket: 51667
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 51665
New client attempting to connect on port: 51668-51665
  Address = /10.128.25.213:51665
This client on port 51668 has uuid 85702317-8035-4b42-8833-e56fa5c06cb5 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-3c8d84e3-5e14-4ad9-b687-7e97b6c33e19, activator-85702317-8035-4b42-8833-e56fa5c06cb5
Taking next connection to: 51665
New client attempting to connect on port: 51678-51665
  Address = /10.128.25.213:51665
Handshake exception on socket: 51678
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 51665
New client attempting to connect on port: 51679-51665
  Address = /10.128.25.213:51665
This client on port 51679 has uuid f8c17bc7-e53f-4749-94d1-abc84acd95fc configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-85702317-8035-4b42-8833-e56fa5c06cb5, play-fork-run-f8c17bc7-e53f-4749-94d1-abc84acd95fc
Taking next connection to: 51665
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51665
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51665
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51665
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51665
